src/acq.c

```diff
@@ -32,8 +33,9 @@ void acq_set_prn(u8 prn)
 {
   chBSemInit(&load_wait_sem, TRUE);
   nap_acq_code_wr_blocking(prn);
-  if (chBSemWaitTimeout(&load_wait_sem, 1000) == RDY_TIMEOUT)
-    printf("acq: Timeout waiting for code load!\n");
+  if (chBSemWaitTimeout(&load_wait_sem, 1000) == RDY_TIMEOUT) {
```

Comment:
nit/lame: it's unrelated to this patchset, but it would be cool to replace the timeout values with a macro reference.